sechub-cli/src/mercedes-benz.com/sechub/util/errorhandler.go

```diff
@@ -9,27 +9,27 @@ import (
 	"os"
 )
 
-// HandleHTTPError handler method for http errors
-func HandleHTTPError(err error, exitcode int) {
+// HandleError - handler method for common errors
+func HandleError(err error, exitcode int) {
```

Comment:
`exitcode` is hard to read. Please indicate that those are two words by changing the name to: `exitCode`, `exit_code` or whatever Go uses.

Reply:
Ok. Changed it everywhere to `exitCode`